mantidproject/mantid
# Fixes to the Inelastic Data Manipulation notes

**Description of work.**

Changes made to `Inelastic Data Manipulation.rst` (was `Indirect Data Manipulation.rst`) as outlined by @SpencerHowells
in [this comment](https://github.com/mantidproject/mantid/pull/35160#issuecomment-1422302773).

This document is important for release 6.6 since it documents the new inelastic interface.

**To test:**

 - Build `docs-html`
 - Check that `Inelastic Data Manipulation.html` displays properly and any changed links work
 - **Important:** I will need someone with specific knowledge to make sure all the changes make sense

*There is no associated issue.* (Just the comment listed above)



---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there is GUI work does it follow the [GUI standards](http://developer.mantidproject.org/Standards/GUIStandards.html)?
- If there are changes in the release notes then do they describe the changes appropriately?
- Are the release notes saved in a separate file, using Issue or PR number for file name and in the correct location?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
